src/DotLiquid.Tests/ExceptionHandlingTests.cs

```diff
@@ -103,6 +105,19 @@ public void TestTimeoutError()
             });
         }
 
+        [Test]
+        public void TestOperationCancelledError()
+        {
+            var template = Template.Parse(" {% for i in (1..1000000) %} {{ i }} {% endfor %} ");
+            var source = new CancellationTokenSource(100);
+            var context = new Context(new List<Hash>(), new Hash(), new Hash(), ErrorsOutputMode.Rethrow, 0, source.Token, CultureInfo.InvariantCulture);

```

Comment:
I recommend to use named parameters when using constant input values such as 0 or null so it is easier to understand for human readers.